releases/mainnet/1.11.0/index.html

```diff
@@ -0,0 +1,27 @@
+<html>
+	<head>
+		<title>Mainnet Release 1.11.0</title>
+	</head>
+	<body>
+		<div>
+			<h2>
+				<strong>Mainnet Release 1.11.0</strong>
+			</h2>
+
+			<ul>
+				<li><a href="/releases/mainnet/1.11.0/contracts.json">Contract addresses</a></li>
+				<li><a href="https://ipfs.io/ipfs/QmVvhd31mz3L92gA1R2qR4SQFLJUyntXPbAAXdedoKn7Jf">Contract addresses (IPFS hosted)</a></li>
+				<li><a href="/releases/mainnet/1.11.0/abi/index.html">ABIs</a></li>
+
+				<li>Links to GitHub repositories
+					<ul>
+						<li>DSS deploy scripts (<a href="https://github.com/makerdao/dss-deploy-scripts/releases/tag/1.11.0" target="_blank">changelog</a>)</li>
+					</ul>
+				</li>
```

Comment:
I think this block should be commented for now as `dss-deploy-scripts` `1.11.0` have not been released yet